generators/app/package-json.js

```diff
@@ -142,7 +143,12 @@ const packageJSON = (current, context) => {
       'clean': 'shx rm -rf ./dist ./test/dist',
       'postclean': 'shx mkdir -p ./dist ./test/dist',
       'lint': 'vjsstandard',
-      'prepublish': 'not-in-install && npm run build && pkg-ok || in-install',
+      'prepublish': 'not-in-install && npm run verify || in-install',
+      'preverify': 'npm run build',
+      'verify': 'npm-run-all -p verify:*',
+      'verify:pkg': 'pkg-ok',
+      'verify:es5': 'es-check es5 \'dist/!(*.es.js|*.css)\'',
+      'verify:es6': 'es-check --module true es6 dist/*.es.js',
```

Comment:
the `es.js` file should be es5 aside from the usage of esmodules.

Also, I find it kind of weird that the flag takes a boolean rather than it being a boolean flag like `--no-module` and `--module`. 🤷‍♀️

Reply:
Yeah its weird I am working on a few issues I found with the author over on the repo.